### DISCUSSION TITLE
Cambio gestore non accedo più da locale

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#3102] [HUB: Home Assistant]
Ciao a tutti, dopo aver cambiato gestore riesco a raggiungere HA solo da duckdns e solo se NON sono in wifi.
Se accedo da casa al ip interno mi esce il seguente errore... qualche idea? Prima del cambio nessun tipo di problema.
![ha_wifi](https://github.com/indomus/forum/assets/79841098/9ddc3bcc-a8e3-43a9-b521-cd2f3658636c)
Grazie in anticipo.

**Comment by snakuzzo** [comment]
Sarà cambiato l'IP che il router assegna al server. Hai verificato?

**Reply by nromagna** [reply]
IP di home assistant? No è sempre lo stesso (fisso), infatti lo pingo tranquillamente.

**Reply by snakuzzo** [reply]
Scusa eh... Apri un browser e accedi da lì `http://ipdelswrver:8123`

**Reply by nromagna** [reply]
Hmmm funziona perfettamente... ma allora (mona io) è solo l'app nel telefono che non  i va... fammi controllare cos'hò combinato.

**Reply by nromagna** [reply]
Re installato app ed è partito... scusatemi ma soprattutto grazie per rapide risposte :)
